packages/core/src/StateMachine.ts

```diff
@@ -252,10 +249,18 @@ export class StateMachine<
    * @param event The received event
    */
   public transition(
-    state: StateValue | State<TContext, TEvent, TResolvedTypesMeta> | undefined,
-    event: Event<TEvent> | SCXML.Event<TEvent>
+    state: StateValue | State<TContext, TEvent, TResolvedTypesMeta> = this
+      .initialState,
+    event: Event<TEvent> | SCXML.Event<TEvent>,
+    actorCtx?: ActorContext<TEvent, State<TContext, TEvent, any>>
```

Comment:
I know that it's quite a pickle to figure out a better name for this - but I really wonder if we couldn't figure out something else for this because it's just so easy to confuse this with `Context` 😢

Reply:
"actorStuff" ?